### PR TITLE
Address doc gen timeouts on Travis

### PR DESCRIPTION
**Pull request by drostron**
.

### CODE REVIEW

File: scripts/build [hunk 1]
```diff
@@ -79,8 +80,16 @@ if [[ "${JOB_NUM##*.}" == "1" ]]; then
...
   # In a fresh JVM process, re-compile without coverage in order to get a jar
-  # that does not contain instrumentation. Generate doc.
-  "$SBT" -DisIsolatedEnv=${ISOLATED_ENV:=false} ++$TRAVIS_SCALA_VERSION $CLEAN 'web/assembly' doc
+  # that does not contain instrumentation.
+  "$SBT" ++$TRAVIS_SCALA_VERSION $CLEAN 'web/assembly'
+
+  # Running `doc` can take awhile, so allow it some extra time when
+  # running on travis to avoid having the build killed due to lack of output.
+  if [[ ${TRAVIS:-} ]]; then
+    travis_wait $TRAVIS_MAX_DOC_MINS "$SBT" ++$TRAVIS_SCALA_VERSION doc
```
Comment (sellout) [inline scripts/build]: Either like here or on the variable definitions, we should have a “FIXME” explaining that `travis_wait` is a hacky workaround.

**Comment by drostron** [comment]
@sellout Ready for another look.

A fix for the CB connection timeout we've been seeing on Travis from time to time and consistently for this PR is included.

**Comment by sellout** [comment]
And … needs a merge from master.

**Comment by drostron** [comment]
@sellout Ready

**Pull request merged**